ui/src/app/app-routing.tsx

```diff
@@ -22,14 +24,21 @@ const signInGuard: Guard = {
   redirectPath: '/login'
 };
 
+const registrationGuard: Guard = {
+  allowed: (): boolean => hasRegisteredAccess(profileStore.get().profile.dataAccessLevel),
+  redirectPath: '/'
```

Comment:
redirecting to homepage seemed most correct, let me know if there's some other thing that seems correcter to you

Reply:
Does it work like this: if an unregistered user navigates to a registration-guarded page, it kicks them back to the homepage?  That sounds reasonable.

Reply:
Yep, exactly. We should probably make a 404 page eventually and do that instead.